examples/poyo/configs/dataset/perich_miller_population_2018.yaml

```diff
@@ -110,4 +116,10 @@
     - m_20140221_random_target_reaching
     - m_20140224_random_target_reaching
   config:
+    weights:
+      movement_periods.random_period: 1.0
+      movement_periods.hold_period: 0.1
+      movement_periods.reach_period: 5.0
+      movement_periods.return_period: 1.0
+      cursor_outlier_segments: 0.0
```

Comment:
Does this work? I think movement_periods.reach_period is not present in the random-target sessions.